docs/ruleset_functions/get_number_of_floors.md

```diff
@@ -0,0 +1,54 @@
+# get_number_of_floors
+**Schema Version:** 0.0.23  
+
+**Description:** gets the number of floors in the building - logic is based on looking at the elevation of zones - zones with an elevation of <0 are considered underground and are not counted.  Parking Garages are not counted
+
+**Inputs:**
+- **zone_input_list** - the list of the zones for which we want to get the number of floors (changed from RMI to this so that we can get the number of floors for a limited list of zones, for example for the function get_hvac_systems_5_6_serving_multiple_floors_b)
```

Comment:
It is my understanding that we need the total number of floors in the building, not number of floors for the BAT.

Reply:
removed the zone_input_list parameter

Reply:
as discussed on RDS call, we will use zone elevation instead of the zone name